continuous-integration/test.sh

```diff
@@ -7,13 +7,13 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )" || exit $?
 cd ~/catkin_ws || exit $?
 
 echo "Installing Zivid API config file"
-install -D "$SCRIPT_DIR"/ZividAPIConfigCPU.yml "$HOME"/.config/Zivid/API/Config.yml || exit $?
+install -D "$SCRIPT_DIR"/ZividAPIConfigComputeDeviceAny.yml "$HOME"/.config/Zivid/API/Config.yml || exit $?
```

Comment:
Can you comment (ideally also in the commit message) why the `ZividAPIConfigCPU` was replaced with `ZividAPIConfigComputeDeviceAny`?

Reply:
It's not an important change. Its just to be even more flexible for the future. No point in saying either CPU or GPU if we in the future run these tests with a discrete GPU.